src/WebJobs.Script.WebHost/Middleware/HttpThrottleMiddleware.cs

```diff
@@ -47,7 +47,7 @@ public async Task Invoke(HttpContext httpContext, IOptions<HttpOptions> httpOpti
             {
                 // only check host status periodically
                 Collection<string> exceededCounters = new Collection<string>();
-                _rejectRequests = await performanceManager.IsUnderHighLoadAsync(exceededCounters);
+                _rejectRequests = performanceManager.PerformanceCountersExceeded(exceededCounters);
```

Comment:
This reverts this check to the behavior prior to https://github.com/Azure/azure-functions-host/commit/826c0b520c1e81d2f3106d08711c08f1bce4aa95 - we'll revert to only checking sandbox counters here (Connections, Threads, Sections, etc.). That logic was long standing.